### ISSUE TITLE
hopper/~in/~out flag doesn't work correctly

### ISSUE THREAD

**Issue by RedstoneFuture** [pop4959/LWCX#144]
**LWC Version (/lwc admin version):** 2.2.6
**Server version (/version):** 1.16.4 PaperMC

**Short five word summary of issue:** 

With my last short test, it seems that the hopper-flag down't block the items-transport correctly.

**Detailed description of problem:**
![2021-01-24_12 21 44](https://user-images.githubusercontent.com/4140635/105628774-36c89e00-5e3f-11eb-8fb7-12948b46b2b3.png)

In my case, I've locked all chests and hopper in the vertical. By default, the "hopper", "hopperin" and "hopperout" flag are defined with "false" and so the items down't have to switch from the first to the second chest. But she does.

Can you please check the basic functionality of this flag? If you can't replicate it, I will re-test it exactly.

paper.yml
```
    hopper:
      cooldown-when-full: true
      disable-move-event: false
```

**If there was an error in console, put it here:**
No.

**Comment by RedstoneFuture** [comment]
EDIT: If I didn't lock the first hopper, the hopper-out-flag will work for the first chest.

**Comment by pop4959** [comment]
The hopper flags are checked per-protection. That means that if you lock hoppers, the hoppers will also need permission. So for example allowing items **out** of the top chest, **in/out** of the hoppers, and **in** to the bottom chest. I usually don't test this configuration, because it's quite a hassle to have to lock and flag every hopper. It's also usually not that important to lock them, since hoppers are rarely used for long-term storage (which is why the default is that they aren't locked).

**Comment by RedstoneFuture** [comment]
Yes, but it doesn't make sense, that I lock all chests and hooper and the items will switch from the first top chest to the bottom chest (first picture).

**Comment by pop4959** [comment]
Tested and appears to be working fine in the configuration you have in your screenshot

**Comment by RedstoneFuture** [comment]
So the items down't switch between the chests in your test? And you locked all 4 blocks, right?

**Comment by Phoenix616** [comment]
Most likely due to the fact that they were all protected by the same owner see https://github.com/pop4959/LWCX/blob/e0ff3ddd19e32f913f149e48803e37db4762665d/src/main/java/com/griefcraft/listeners/LWCPlayerListener.java#L524-L527

**Comment by pop4959** [comment]
Thanks for catching that Phoenix, it looks like this is just a misunderstanding then. The behavior should be correct, since the hopper flags are intended to be used to limit other players interaction with your protections, and not as a general purpose mechanic.

**Comment by RedstoneFuture** [comment]
Update: With version `2.2.7 #150` it works for my at now. Maybe was it fixed with the new updates since 2.2.6 (last version promoted in spigot) or I fixed an own mistake.
